sklearn/utils/multiclass.py

```diff
@@ -248,6 +248,11 @@ def type_of_target(y, input_name=""):
 
         .. versionadded:: 1.1.0
 
+    raise_unknown : bool, default=False
+        If True, raise an error when the target type is unknown.
```

Comment:
It would be beneficial to mention what are "types" we do know, or link to the relevant doc where it lives.